.github/workflows/license_finder.yml

```diff
@@ -36,7 +36,9 @@ jobs:
       - name: Generate requirements.txt (exclude dev dependencies)
         run: |
           poetry export -f requirements.txt --without-hashes > requirements.txt
+          pip install -r requirements.txt
 
       - name: Run license finder
         run: |
-          poetry run license_finder --python-version=3
+          /opt/hostedtoolcache/Ruby/*/x64/lib/ruby/gems/*/gems/license_finder-*/bin/license_finder_pip.py requirements.txt
```

Comment:
We still need this command to check license finder and fail the workflow if needed. Running license finder itself returns an exit status of 0 even if it's failing.